### PR TITLE
feature: Dubbo api docs UI

### PR DESCRIPTION
**Pull request by KeRan213539**
feature: Dubbo api docs

### CODE REVIEW

File: dubbo-admin-server/pom.xml [hunk 1]
```diff
@@ -172,10 +173,30 @@
             <version>${mockito-version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
```
Comment (beiwei30) [inline dubbo-admin-server/pom.xml:180]: pls. don't use lombok

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/ApiDocsController.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * dubbo doc ui server api.
+ * @author klw(213539@qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/ApiDocsController.java:54]: pls. remove author info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/beans/DubboApiDocsParamInfoBean.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package org.apache.dubbo.admin.controller.beans;
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/beans/DubboApiDocsParamInfoBean.java:1]: pls. add apache license header.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/beans/DubboApiDocsParamInfoBean.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * api parameter bean.
+ *
+ * @author klw(213539 @ qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/beans/DubboApiDocsParamInfoBean.java:10]: again, pls. remove author info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/editors/CustomLocalDateEditor.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+import org.apache.dubbo.admin.utils.LocalDateTimeUtil;
+
+import java.beans.PropertyEditorSupport;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Localdate editor for controller.
+ *
+ * @author klw(213539 @ qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/editors/CustomLocalDateEditor.java:29]: pls. remove author info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/editors/CustomLocalDateTimeEditor.java [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+import org.apache.dubbo.admin.utils.LocalDateTimeUtil;
+
+import java.beans.PropertyEditorSupport;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * LocalDateTime editor for controller.
+ *
+ * @author klw(213539 @ qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/editors/CustomLocalDateTimeEditor.java:29]: pls. remove author info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/model/dto/docs/ApiInfoRequest.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+ */
+package org.apache.dubbo.admin.model.dto.docs;
+
+import io.swagger.annotations.ApiParam;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Obtain the API module list and the request parameters of the API parameter information interface.
+ * @author klw(213539@qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/model/dto/docs/ApiInfoRequest.java:25]: pls. remove author info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/model/dto/docs/CallDubboServiceRequest.java [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+package org.apache.dubbo.admin.model.dto.docs;
+
+import io.swagger.annotations.ApiParam;
+import lombok.Getter;
+import lombok.Setter;
+
+
+/**
+ * Call Dubbo api to request parameters.
+ * @author klw(213539@qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/model/dto/docs/CallDubboServiceRequest.java:26]: remove athor info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/model/dto/docs/CallDubboServiceRequestInterfaceParam.java [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+ */
+package org.apache.dubbo.admin.model.dto.docs;
+
+import io.swagger.annotations.ApiParam;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Parameters passed to duboo service api.
+ * @author klw(213539@qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/model/dto/docs/CallDubboServiceRequestInterfaceParam.java:25]: remove author info.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/utils/LocalDateTimeUtil.java [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Date time tool class of LocalDateTime.
+ *
+ * @author klw(213539 @ qq.com)
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/utils/LocalDateTimeUtil.java:27]: remove author info.

File: dubbo-admin-server/src/main/resources/application-test.properties [hunk 11]
```diff
@@ -16,7 +16,7 @@
 #
 
 # centers in dubbo2.7
-admin.registry.address=zookeeper://127.0.0.1:2183
-admin.config-center=zookeeper://127.0.0.1:2183
-admin.metadata.address=zookeeper://127.0.0.1:2183
+admin.registry.address=nacos://127.0.0.1:8848
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/resources/application-test.properties:19]: pls. rollback to zookeeper

File: dubbo-admin-server/src/main/resources/application.properties [hunk 12]
```diff
@@ -16,9 +16,9 @@
 #
 
 # centers in dubbo2.7
-admin.registry.address=zookeeper://127.0.0.1:2181
-admin.config-center=zookeeper://127.0.0.1:2181
-admin.metadata-report.address=zookeeper://127.0.0.1:2181
+admin.registry.address=nacos://127.0.0.1:8848
```
Comment (beiwei30) [inline dubbo-admin-server/src/main/resources/application.properties:19]: pls. rollback to zookeeper since I believe it's irrelevant to this PR

**Review by beiwei30 — CHANGES_REQUESTED** [review]

**Review by beiwei30 — APPROVED** [review]
LGTM.

**Review by m7adf7d — COMMENTED** [review]
m7adf7d-admin-backend/src/main/java/org/apache/m7adf7d/admin/service/impl/UserServiceImpl.java

**Pull request merged**